R/dots.R

```diff
@@ -1,23 +1,70 @@
-#' Extract dots with splicing semantics
+#' Extract tidy dots
```

Comment:
"Capture dots, tidily" ?

(Although we've used capture to imply unevaluated in other places)

Reply:
Yes I think "extract" is better than "capture" to avoid confusion.

Reply:
But what are we extracting them from? How about collect?

Reply:
From the `...` pairlist of arguments :)

"collect" sounds good to me.